.circleci/config.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - run:
           name: package apisx
           command: |
-            make package type=deb app=apisix version=${TEST_VERSION} checkout=master image_base=ubuntu image_tag=20.04
+            make package type=deb app=apisix version=${TEST_VERSION} checkout=revolyssup/openssl3 image_base=ubuntu image_tag=20.04
```

Comment:
Is it possible to build from the openssl3 branch of apisix? This means you may merge code from your personal branch into the openssl3 branch

Reply:
There is kind of circular dependency here because this repo clones apisix in its ci and apisix clones this repo in its ci.
Since APISIX checks out this repo in its own CI, I am waiting for this PR to be reviewed to then make changes on the openssl branch on APISIX. So the steps can be as follows:
1. Review this PR and the only changes made to be left will be to point the CI here to openssl branch in APISIX.
2. I will make sure the openssl branch in APISIX has corresponding changes with this PR. And then get the PR merged in APISIX openssl branch (Note that at this point this merged PR will have reference to my fork)
3. I will point the CI here to apisix upstream openssl branch so that this PR can be approved and merged.
4. Then I will create a new PR in apisix openssl branch making sure that all references to my fork are replaced with the upstream build tools repo.
5. At this point we will have openssl changes merged in both repos.

What do you think? @soulbird

Reply:
ok